packages/react/src/CTABanner/CTABanner.tsx

```diff
@@ -0,0 +1,93 @@
+import React, {forwardRef, type Ref} from 'react'
+import clsx from 'clsx'
+import type {BaseProps} from '../component-helpers'
+import {Heading} from '../Heading'
+import {Text} from '../Text'
+
+/**
+ * Design tokens
+ */
+import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/cta-banner/colors-with-modes.css'
+
+/** * Main Stylesheet (as a CSS Module) */
+import styles from './CTABanner.module.css'
+
+export type CTABannerProps = BaseProps<HTMLDivElement> & {
+  children: React.ReactNode | React.ReactNode[]
+  align?: 'start' | 'center'
+  hasBorder?: boolean
+  hasShadow?: boolean
+  hasBackground?: boolean
+}
+
+export const Root = forwardRef(
+  (
+    {
+      align = 'start',
+      hasBorder = false,
+      hasShadow = true,
+      hasBackground = true,
+      className,
+      children,
+      ...props
+    }: CTABannerProps,
+    ref: Ref<HTMLDivElement>
+  ) => {
+    return (
+      <section ref={ref} className={styles.CTABanner} {...props}>
+        {hasShadow && (
+          <>
+            <div aria-hidden className={clsx(styles['CTABanner--Shadow'], styles['CTABanner--StartShadow'])} />
+            <div aria-hidden className={clsx(styles['CTABanner--Shadow'], styles['CTABanner--EndShadow'])} />
```

Comment:
```suggestion
            <div aria-hidden className={clsx(styles['CTABanner-shadow'], styles['CTABanner-shadow--end'])} />
```

Reply:
This may be different in next push if we go with the generic box shadow